.github/workflows/integration-tests.yml

```diff
@@ -96,121 +93,4 @@ jobs:
-            if grep -r "crashed" "$zkvm_folder"/*.json 2>/dev/null; then
-              echo "Found 'crashed' string in ZKVM folder JSON files"
-              exit 1
-            fi
-          fi
-      
-      - name: Sanitize artifact filenames
-        run: |
-          find zkevm-metrics -depth -name '*:*' | while IFS= read -r file; do
-            mv -- "$file" "$(echo "$file" | tr ':' '_')"
-          done
-
-      - name: Upload metrics artifact
-        uses: actions/upload-artifact@v4
-        with:
-          name: zkevm-metrics-stateless-validator-${{ matrix.test-type }}-${{ matrix.zkvm }}-${{ matrix.action }}
-          path: zkevm-metrics
-          retention-days: 14
-
-  empty-program:
-    name: "Empty Program (${{ matrix.zkvm }} / ${{ matrix.action }})"
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        zkvm: [sp1, risc0, zisk]
-        action: [execute, prove] 
-        exclude:
-          - zkvm: zisk # Setup too complex for CI and ere image doesn't bake big proving key 
-            action: prove  
-
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
-
-      - name: Install Rust toolchain
-        uses: dtolnay/rust-toolchain@nightly
-
-      - name: Pull ere images
-        run: |
-          for variant in "" "-${{ matrix.zkvm }}"; do
-            src="ghcr.io/eth-act/ere/ere-base${variant}:${ERE_TAG}"
-            dst="ere-base${variant}:${ERE_TAG}"
-            docker pull "$src"
-            docker tag "$src" "$dst"
-          done
-
-      - name: Run benchmark
-        run: RUST_LOG=info cargo run -p ere-hosts --release -- --zkvms ${{ matrix.zkvm }} --action ${{ matrix.action }} empty-program
-
-      - name: Validate metrics output
-        run: |
-          find ./zkevm-metrics
-          if [ ! -d "./zkevm-metrics" ]; then
-            echo "zkevm-metrics folder does not exist"
-            exit 1
-          fi
-          if [ ! -f "./zkevm-metrics/hardware.json" ]; then
-            echo "hardware.json file does not exist in zkevm-metrics folder"
-            exit 1
-          fi
-          
-          zkvm_folder=$(find ./zkevm-metrics -type d -name "${{ matrix.zkvm }}-*" | head -n 1)
-          if [ -z "$zkvm_folder" ]; then
-            echo "No ${{ matrix.zkvm }} folder found in zkevm-metrics"
-            exit 1
-          fi
-          
-          if [ ! -f "$zkvm_folder/empty_program.json" ]; then
-            echo "empty_program.json file does not exist in $zkvm_folder"
-            exit 1
-          fi
-
-      - name: Upload metrics artifact
-        if: always()
-        uses: actions/upload-artifact@v4
-        with:
-          name: zkevm-metrics-empty-program-${{ matrix.zkvm }}-${{ matrix.action }}
-          path: zkevm-metrics
-          retention-days: 14
+        run: CI=1 RUST_LOG=info,sp1_core_executor=warn ZKVM=${{ matrix.zkvm }} cargo test --release -p integration-tests -- --test-threads=1 ${{ matrix.test }}
```

Comment:
TL;DR here:
- `CI=1` is to pipe the expected env that ZisK expects when you only use CPU as a proving resource. If we don't do this, the build-args change and it tries to rebuild the pulled ere image to compile NVIDIA libraries which we don't use in CI (thus why the env name).
- `ZKVM` is an env var in the new integration test crate which allows to filter which zkvms are run. If you run locally without setting this env-var, it will run for all zkvms that make sense. The logic of using this env var is to run stuff more in parallel only in CI, and also identify quicker which zkvm might be failing (since each one here is a matrix combination)
- `--test-threads=1` is to avoid clashes because of zkvm patches.